telebot/handler_backends.py

```diff
@@ -265,6 +261,15 @@ class CancelUpdate:
     Update will skip handler and execution
     of post_process in middlewares.
     """
+    def __init__(self) -> None:
+        pass
+
 
+class ContinueHandling:
+    """
+    Class for continue updates in handlers.
+    Just return instance of this class 
+    in handlers to continue process.
```

Comment:
Add code block here, something like this:

```suggestion
    in handlers to continue process.
    
    .. code-block:: python3
        :caption: Example of using ContinueHandling

        @bot.message_handler(commands=['start'])
        def start(message):
            bot.send_message(message.chat.id, 'Hello World!')
            return ContinueHandling()
        
        @bot.message_handler(commands=['start'])
        def start2(message):
            bot.send_message(message.chat.id, 'Hello World2!')
    
```

Same with async one but with async syntax.